src/translations/en.json

```diff
@@ -981,6 +986,7 @@
     "EquipmentCopied": "Data successfully copied from equipment {equipmentId}",
     "EquipmentCopyFailed404": "Equipment copy failed: the equipment was not found",
     "EquipmentCopyFailed": "Equipment copy failed",
+    "equipmentType": "Equipment type",
```

Comment:
there is already an EquipmentType translation key for this.

Reply:
done